src/components/MultipleHeadlines.vue

```diff
@@ -12,7 +12,7 @@
   // Define props for the component
   const { count = 4, articleNumbers = [0,1,2,3] } = defineProps<{
     count: number,
-    articleNumbers: number
+    articleNumbers: number | number[]
```

Comment:
Is this just saying "either a single number, or if maybe an array of numbers is returned?

Reply:
Yeah